Adyen/Model/Nexo/PaymentInstrumentData.cs

```diff
@@ -23,5 +22,13 @@ public partial class PaymentInstrumentData
         /// <remarks/>
         [System.Xml.Serialization.XmlAttributeAttribute()]
         public PaymentInstrumentType PaymentInstrumentType;
+
+        /// <remarks/>
+        [System.Xml.Serialization.XmlAttributeAttribute()]
+        public StoredValueAccountID StoredValueAccountID;
+
+        /// <remarks/>
```

Comment:
can we remove this ?

Reply:
It is mentioned in the nexo documentation

Reply:
remarks comments are auto generated by the xsd to models conversion